go/vt/mysqlctl/backupengine.go

```diff
@@ -109,6 +112,16 @@ type BackupRestoreEngine interface {
 // BackupEngine and RestoreEngine.
 var BackupRestoreEngineMap = make(map[string]BackupRestoreEngine)
 
+func init() {
+	for _, cmd := range []string{"mysqlctl", "mysqlctld", "vtadmin", "vtbackup", "vtcombo", "vtctl", "vtctld", "vttablet", "vttestserver"} {
```

Comment:
same comment r.e. tracing (and probably every other file in this package)

Reply:
I have tried my best to update this list .. besides I am assuming if I have forgot any binary then some of our test would high likely break, right ?